oat-sa/extension-tao-testqti
# Fix/tao 6416/inject theme namespace on test init

Depends on oat-sa/tao-core#1775 && oat-sa/extension-tao-delivery-rdf#208 && oat-sa/extension-tao-act#1037 && oat-sa/tao-core#1777 && oat-sa/extension-tao-itemqti#1172
Basically, the problem is that on non-LTI launches theme set on delivery edit form isn't applied to items.
To test it:
* create a delivery on a project which uses theme-switcher/has option to switch theme on delivery edit form;
* launch the delivery
* see if the item theme got applied
